jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiControllerTest.java

```diff
@@ -50,16 +52,18 @@
 @ExtendWith(MockitoExtension.class)
 @SpringBootTest
 @Transactional
-@ContextConfiguration(locations = "/test-context.xml")
+@ContextConfiguration(locations = "/test-context.xml", classes = {ObjectMapperConfig.class})
```

Comment:
Should we be mixing the context configuration i.e. should we just use classes or just use xml.

Reply:
According to Spring documentation XML and Java config are not exclusive, so they can be mixed.

The only reason I imported object mapper config separately into the test, is to allow flexibility for other situations where only the test-context.xml is needed, Otherwise I could have added it inside test-context.xml.

Maybe I am overthinking it. So I could add the Object Mapper config inside test.context.xml as follows:
`<bean class="uk.gov.homeoffice.digital.sas.jparest.config.ObjectMapperConfig"/>`
and remove `classes = {ObjectMapperConfig.class}` from `@ContextConfiguraton`

What do you think?

Reply:
The reference to mixing was with respect to our approach i.e. let's chose one or the other. I quite like the classes option because I feel it's a little more visible but then that makes it more verbose. 
If we never vary it, maybe we can just add to the xml as per your suggestion.

Reply:
I’ve incorporated the object mapper config in the xml test context